dpnp/__init__.py

```diff
@@ -26,6 +26,7 @@
 
 import os
 import sys
+from importlib import import_module
```

Comment:
Why do we need that?

Reply:
I forgot to delete it when I used lazy import
Thanks